### ISSUE TITLE
useWebsocket in webworker：window in not defined

### ISSUE THREAD

**Issue by sy19971224** [vueuse/vueuse#3937]
### Describe the bug

in `work.js`，i used `useWebsocket`，the socket  connect successfully，but when i want to use the websocket
response data ,  an error message appears ：  `window in not defined`
here is my `worker.js`
`import {useWebsocket} from @vueuse/core
 self.addEventlistener('message',(e) => {
    const {data} = useWebsocket(e.data.url,{
       autoReconnect:{delay:3000},
       onConnected:(ws) => {
         const sub = JSON.stringify({xxxx})
         ws.send(sub)
       }
    })
    // ----- error here : console.log(data) ---- 
 })
`

### Reproduction

none

### System Info

```Shell
npm：7.21.1
vue：2.7
vite：^3.1.3
vueuse/core:^7.5.3
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sy19971224** [comment]
```
import {useWebsocket} from '@vueuse/core'
 self.addEventlistener('message',(e) => { 
 const {data} = useWebsocket(e.data.url,{ 
  autoReconnect:{delay:3000},
  onConnected:(ws) => { 
     const sub = JSON.stringify({xxxx}) 
     ws.send(sub) 
      } 
   }) 
   // ----- error here : console.log(data) ---- 
}) 
```

**Comment by ilyaliao** [comment]
Looks like it's been fixed by #4229. I tested and found no issues. Feel free to reopen if it's not fixed.
